fastlane/README.md

```diff
@@ -13,47 +13,66 @@ For _fastlane_ installation instructions, see [Installing _fastlane_](https://do
 
 # Available Actions
 
-## Android
+### bump
 
-### android test
+```sh
+[bundle exec] fastlane bump
+```
+
+Bump version, edit changelog, and create pull request
```

Comment:
Note that this now does much more than before. Besides asking devs for a version number and updating that version number in the necessary places, this will also prompt devs to edit the CHANGELOG as part of the process, will create a release branch, commit changes, push it and create a PR. ❤️ automation!